cli/userlist.go

```diff
@@ -111,13 +111,13 @@ func userSingle() *cobra.Command {
 }
 
 func displayUser(ctx context.Context, stderr io.Writer, client *codersdk.Client, user codersdk.User) string {
-	tableWriter := cliui.Table()
+	tw := cliui.Table()
```

Comment:
Is there a specific reason we're not using `cliui.DisplayTable` here? (I did notice the logic is fairly complex here; so perhaps that's why.)

Reply:
This is a vertical table instead of a horizontal table which is not supported by the generic renderer atm and only used in two places. We can add support for vertical tables in the future.

Reply:
Ahh, yeah that makes sense; thanks for clarifying!